datadog_checks_base/datadog_checks/base/utils/headers.py

```diff
@@ -31,6 +49,7 @@ def headers(agentConfig, **kwargs):
     # Build the request headers
     version = __get_version(agentConfig)
     res = {'User-Agent': 'Datadog Agent/{}'.format(version)}
+    res.update(_get_common_headers())
```

Comment:
Isn't this a dupe of `get_default_headers()`?

Reply:
Well, kind of, and that's bugging me a bit, but I think we should treat that as a separate PR. Some integrations are using it, and we'd need some separate work to update them to use `get_default_headers()`, which is purely refactoring. This PR focuses on the behavioral change wrt headers. Sounds legit?

Reply:
I was more meaning you could do 
```suggestion
    res = get_default_headers()
```